quickstarts/microsoft.databricks/databricks-all-in-one-template-for-vnet-injection-with-load-balancer/azuredeploy.json

```diff
@@ -234,6 +235,7 @@
               "networkSecurityGroup": {
                 "id": "[variables('nsgId')]"
               },
+              "defaultOutboundAccess": false,
```

Comment:
You don't need the following as dependencies for workspace. Because workspace depends on vNet, vNet depends on NSG. LB depends on Public IP all of them are correctly linked. So we can remove this

        "[resourceId('Microsoft.Network/loadBalancers', parameters('loadBalancerName'))]",
        "[resourceId('Microsoft.Network/networkSecurityGroups', parameters('nsgName'))]",